### PR TITLE
Changed windows VM version

### LINKED ISSUE #479: Releasing on windows fails
The job to release the windows version of the software is failing on the signature step.

https://github.com/vegaprotocol/vegawallet/runs/5107117443?check_suite_focus=true

### PR DESCRIPTION
**Pull request by peterbarrow**
Downgraded Windows VM version so that the code signer will still work.
Closes #479

### CODE REVIEW

File: .github/workflows/release-binaries.yml [hunk 1]
```diff
@@ -195,7 +195,7 @@ jobs:
   release-windows:
     needs: check-version
     name: Release on Windows ${{ matrix.arch }}
-    runs-on: windows-latest
+    runs-on: windows-2019
```
Comment (jeremyletang) [inline .github/workflows/release-binaries.yml:198]: Why is this needed?

Comment (peterbarrow) [reply .github/workflows/release-binaries.yml:198]: See above "Downgraded Windows VM version so that the code signer will still work."

Comment (jeremyletang) [reply .github/workflows/release-binaries.yml:198]: ah didn't read the description. OK.

**Review by jeremyletang — APPROVED** [review]
